server/appservices/oauth2.go

```diff
@@ -53,13 +66,17 @@ func (a *AppServices) StoreOAuth2User(r *incoming.Request, appID apps.AppID, act
 	if err != nil {
 		return err
 	}
+	if bytes.Equal(data, oldData) {
+		return nil
+	}
```

Comment:
So the main change here is that we don't call `a.store.OAuth2.SaveUser` if the `OAuth2User` has not changed. Is that the main fix in the PR?

Reply:
No, the main fix is adding the refresh bindsings websocket message to SaveOAuth2App